tests/test_run.py

```diff
@@ -0,0 +1,35 @@
+from mock import patch
+
+from scripts.c2r_script import run_convert2rhel
+
+
+@patch("scripts.c2r_script.SCRIPT_TYPE", "CONVERSION")
+@patch("scripts.c2r_script.IS_CONVERSION", True)
```

Comment:
We probably don't need this patch here... The `IS_CONVERSION` constant will be converted to True because of the `SCRIPT_TYPE`.